lib/fishjam/room_service.ex

```diff
@@ -148,7 +150,13 @@ defmodule Fishjam.RoomService do
       {:reply, {:ok, room, Fishjam.address()}, state}
     else
       {:error, :room_already_exists} = error ->
+        Logger.warning("Room creation failed, because it already exists")
+
         {:reply, error, state}
+
+      reason ->
+        Logger.warning("Room creation failed with reason: #{inspect(reason)}")
+        {:reply, :room_doesnt_start, state}
```

Comment:
1. Shouldn't this be `{:error, :room_doesnt_start}`?
2. This needs to be handled in `RoomController` as well